NeuralAmpModeler/NeuralAmpModeler.xcworkspace/contents.xcworkspacedata

```diff
@@ -1,12 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Workspace
    version = "1.0">
-   <FileRef
```

Comment:
This looks like a problem? You're removing these source files from the project?

Reply:
Apologies for the premature comment while I refresh myself on what each button does on GitHub 😅

Reply:
I removed them from the xcode workspace... they should be in the xcode projects individually